roles/ipabackup/README.md

```diff
@@ -0,0 +1,331 @@
+ipabackup role
+==============
+
+Description
+-----------
+
+This role allows to backup an IPA server, to copy a backup from the server to the controller, to copy all backups from the server to the controller, to remove a backup from the server, to remove all backups from the server, to restore an IPA server locally and from the controller and also to copy a backup from the controller to the server.
```

Comment:
I'd change `IPA server locally and from the controller` to `IPA server from backup data in the server or on the controller,`